ycsb/main.go

```diff
@@ -74,7 +76,7 @@ var writeDuration = flag.Duration("write-duration", 0,
 var verbose = flag.Bool("v", false, "Print *verbose debug output")
 var drop = flag.Bool("drop", true,
 	"Drop the existing table and recreate it to start from scratch")
-var rateLimit = flag.Uint64("rate-limit", 0,
+var maxRate = flag.Uint64("rate-limit", 0,
 	"Maximum number of operations per second per worker. Set to zero for no rate limit")
```

Comment:
In `kv`, the rate limit is global, not per worker. We should probably do the same here for consistency. I think you just have to pass in a shared `*rate.Limiter` to `newYcsbWorker`.

Reply:
+1

Reply:
Done.